### PR TITLE
Cleaning up a bunch of codeCommit artifacts

### PR DESCRIPTION
**Pull request by CallMeCCLemon**

### CODE REVIEW

File: lib/PipelineStack.ts [hunk 1]
```diff
@@ -4,35 +4,18 @@ import * as s3 from "aws-cdk-lib/aws-s3";
 import * as codePipeline from "aws-cdk-lib/aws-codepipeline";
 import * as codeCommit from 'aws-cdk-lib/aws-codecommit';
 import * as codePipelineActions from "aws-cdk-lib/aws-codepipeline-actions";
-import * as codeCommitActions from "aws-cdk-lib/aws-codepipeline-actions";
 import * as secrets from "aws-cdk-lib/aws-secretsmanager";
-import * as cf from "aws-cdk-lib/aws-cloudfront";
 import {Construct} from "constructs";
+import * as cf from "aws-cdk-lib/aws-cloudfront";
```
Comment (Tonipenyallop) [inline lib/PipelineStack.ts:9]: I think move this line above would be better for unite the style

Comment (CallMeCCLemon) [reply lib/PipelineStack.ts:9]: Sure thing. :+1:

Comment (CallMeCCLemon) [reply lib/PipelineStack.ts:9]: Will handle this in the next PR.

**Review by Tonipenyallop — APPROVED** [review]
Code looks good and you are going to fix next PR for the import line so I will provide you stamp

**Pull request merged**